### PR TITLE
New Swahili Cheat Sheet

### PR DESCRIPTION
**Pull request by abadojack**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [* ] New Instant Answer
    - [ *] Cheat Sheets: **`New Swahili Cheat Sheet`**
    
###### Description of changes

Creates new Swahili cheat sheet.

---

Instant Answer Page: https://duck.co/ia/view/swahili_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/language/swahili.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
+{
+   "id" : "swahili_cheat_sheet",
+   "name" : "Swahili Cheat Sheet",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/language/swahili.json]: You can remove `Cheat Sheet` from the name. And could you move up the `description` field to be below `name`? It'll help quick readability of the JSON. Thanks!

Comment (abadojack) [inline share/goodie/cheat_sheets/json/language/swahili.json]: @gaulrobe Done.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=swahili cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/swahili_cheat_sheet_mobile.png?iJlYhtN1z3)

File: share/goodie/cheat_sheets/json/language/swahili.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+         },
+         {
+            "val" : "Nzuri",
+            "key" : "I'm fine (Reply to 'How are you?')"
+         },
+         {
+            "val" : "Hujambo",
+            "key" : "Hello"
+         },
+		 {
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/language/swahili.json]: Could you get rid of the extra tabbing in this block? Thanks!

Comment (abadojack) [inline share/goodie/cheat_sheets/json/language/swahili.json]: Done.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=swahili cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/swahili_cheat_sheet_mobile.png?SVeguE04Gn)

**Comment by tagawa** [comment]
Thanks for making another Instant Answer @abadojack! Are you a native speaker of Swahili or would you like us to put a call out for language reviewers?

**Comment by abadojack** [comment]
@tagawa  Yes, I'm a native speaker of Swahili.

**Comment by tagawa** [comment]
@abadojack Great. It looks good to me on the beta server so let's merge this.
Thanks for making another Instant Answer! 🍰

**Comment by daxtheduck** [comment]
@abadojack this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by abadojack** [comment]
@tagawa You're welcome.

**Pull request merged**